sdk/include/opentelemetry/sdk/logs/logger_provider.h

```diff
@@ -6,16 +6,13 @@
 
 #  include <memory>
 #  include <mutex>
-#  include <string>
 #  include <vector>
 
 #  include "opentelemetry/logs/logger_provider.h"
-#  include "opentelemetry/logs/noop.h"
 #  include "opentelemetry/nostd/shared_ptr.h"
-#  include "opentelemetry/sdk/common/atomic_shared_ptr.h"
-#  include "opentelemetry/sdk/logs/logger.h"
-#  include "opentelemetry/sdk/logs/logger_context.h"
-#  include "opentelemetry/sdk/logs/processor.h"
+#  include "opentelemetry/nostd/string_view.h"
+#  include "opentelemetry/sdk/resource/resource.h"
+#  include "opentelemetry/version.h"
```

Comment:
This file has reference to `KeyValueIterable`, - seems it is indirectly included.  Not a blocker, we can fix this separately.